src/services/script_patcher/script_patcher_service.cpp

```diff
@@ -14,6 +14,9 @@ namespace big
 
 	script_patcher_service::~script_patcher_service()
 	{
+		for (auto& patch : m_script_patches)
+			patch.disable(get_data_for_script(patch.get_script()));
+
```

Comment:
This is the same code pending from #2967. It hasn't been merged, as it currently doesn't solve the issue at hand.